### PR TITLE
[docs] Documentation proofreading.

### PR DESCRIPTION
**Pull request by VV-corrector**
Documentation proofreading.

### CODE REVIEW

File: modules/110-istio/docs/README.md [hunk 1]
```diff
@@ -1,10 +1,10 @@
 ---
-title: "The istio module"
+title: "The Istio module"
```
Comment (VV-corrector) [inline modules/110-istio/docs/README.md:2]:
```suggestion
title: "The istio module"
```

File: modules/110-istio/docs/README.md [hunk 2]
```diff
@@ -1,10 +1,10 @@
...
 webIfaces:
-- name: istio
+- name: Istio
```
Comment (VV-corrector) [inline modules/110-istio/docs/README.md:4]:
```suggestion
- name: istio
```

File: modules/110-istio/docs/README.md [hunk 3]
```diff
@@ -1,10 +1,10 @@
...
 ---
 
-Currently supported istio versions:
+Currently supported Istio versions:
```
Comment (VV-corrector) [inline modules/110-istio/docs/README.md:7]:
```suggestion
Currently supported istio versions:
```

File: modules/110-istio/docs/README_RU.md [hunk 4]
```diff
@@ -1,13 +1,13 @@
 ---
-title: "Модуль istio"
+title: "Модуль Istio"
```
Comment (VV-corrector) [inline modules/110-istio/docs/README_RU.md:2]:
```suggestion
title: "Модуль istio"
```

File: modules/110-istio/docs/README_RU.md [hunk 5]
```diff
@@ -1,13 +1,13 @@
...
 webIfaces:
-- name: istio
+- name: Istio
```
Comment (VV-corrector) [inline modules/110-istio/docs/README_RU.md:4]:
```suggestion
- name: istio
```

File: modules/110-istio/docs/README_RU.md [hunk 6]
```diff
@@ -1,13 +1,13 @@
...
 ---
 
-На данный момент поддерживаются следующие версии istio:
-* 1.12 (устарела, скоро будет снята с поддержки)
-* 1.13 (устарела, скоро будет снята с поддержки)
-* 1.16
+На данный момент поддерживаются следующие версии Istio:
```
Comment (VV-corrector) [inline modules/110-istio/docs/README_RU.md:7]:
```suggestion
На данный момент поддерживаются следующие версии istio:
```

**Comment by deckhouse-BOaTswain** [comment]
Cherry pick PR [6043](https://github.com/deckhouse/deckhouse/pull/6043) to the branch [release-1.52](https://github.com/deckhouse/deckhouse/tree/release-1.52) successful!

**Pull request merged**